Octokit.Reactive/Clients/IObservableUsersClient.cs

```diff
@@ -53,6 +53,15 @@ public interface IObservableUsersClient
         IObservableUserKeysClient Keys { get; }
 
         /// <summary>
+        /// A client for GitHub's UserUser GPG Keys API.
+        /// </summary>
+        /// <remarks>
+        /// See the <a href="https://developer.github.com/v3/users/gpg_keys/">User GPG Keys documentation</a> for more information.
+        /// </remarks>
+        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Gpg")]
+        IObservableUserGpgKeysClient GpgKeys { get; }
```

Comment:
While you were away, I believe it was decided to move towards "singular" property names for sub clients (but plaralised class names) - one such mention is in #1207 .  

So for any new clients being added we should probably follow this convention, thus `IObservableUserGpgKeysClient GpgKey { get; }` (and `IUserGpgKeysClient GpgKey { get; }` on `IUsersClient`)